srcs/server/Config.cpp

```diff
@@ -1,8 +1,7 @@
 #include "Config.hpp"
 
 Config::Config() {
-	_client_max_body_size = 1024; // in the future is going to come from the config file
-	_listening_port = "3491";
```

Comment:
The value of this variable needs to be set somewhere if not the binary does not run properly